src/util/strconv.h

```diff
@@ -56,4 +57,13 @@ static int ParseBoolean(const char *arg, int *res) {
   return 0;
 }
 
+static char *strtolower(char *str) {
```

Comment:
Note that this is only good for ANSI. When we have the means to handle Unicode (i.e. C++) we should handle this a little bit differently. See https://stackoverflow.com/questions/34433380/lowercase-of-unicode-character.
We can drop a TODO comment, just so we won't forget.

Reply:
@rafie this deserves a separate issue